source/style-guide/style/code-examples.txt

```diff
@@ -10,14 +10,27 @@ Code Examples
    :depth: 1
    :class: singlecol
 
-Use the following guidelines when creating blocks of code as input
-or output examples:
+Use the following guidelines when creating code examples:
 
 -  Don't use screenshots to show code examples. Format them as blocks
    of code by using the appropriate markup in your authoring tool. For
    more information about formatting, see :ref:`text-formatting`.
 
--  When showing input, always include a command prompt (such as $).
+-  Ensure that any placeholder text in code is obvious.
+
+   - Enclose placeholders in angle brackets.
```

Comment:
```suggestion
   - Use all caps for place holder text, with underscores to separate the multiword placeholders. 
```
Angle brackets can be a bit confusing for languages that use angle brackets (for example, React Native uses angle brackets for components). My team currently uses all caps with underscores to denote placeholder text like in this section for `APP_ID`: https://www.mongodb.com/docs/atlas/device-sdks/sdk/react-native/manage-users/authenticate-users/#set-up--realm-react-providers. What do you think?

Reply:
@lindseymoore, yes, I've seen out team using curly braces. I'll make this a bit more flexible.

Reply:
In the example I sent, curly braces are also used as syntax for passing parameters (not as a placeholder indicator). My team does use curly brackets and angled brackets as placeholders for endpoint calls and curl requests:
https://www.mongodb.com/docs/atlas/app-services/sync/configure/enable-sync/#select-a-cluster-to-sync

Perhaps it would be just be good to add a caveat like "where appropriate". I also think that using all caps or capital letters is a good indicator for a placeholder because it is easier to see than lowercase if someone if scanning for a placeholder. I'll suggest another edit!

Reply:
@lindseymoore, since we seem to have so much variation on this, I'm going to stick with just the main line and this can get ironed down the the line.